### PR TITLE
new intro page

### PR DESCRIPTION
**Pull request by enkore**
Inspired by the http://zeromq.org intro page. I think a follow-up to something similar to "Join the Community" would be appropriate (replacing "Fund...").

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,132 @@
-<html>
+ <!DOCTYPE html>
```
Comment (ThomasWaldmann) [inline index.html]: there's a blank at the left.

File: index.html [hunk 2]
```diff
@@ -1,11 +1,132 @@
...
+	</header>
+
+	<main>
+		<h2 id="tagline">Deduplicating archiver<br>with compression and encryption</h2>
+
+		Borg Backup (short: Borg) gives you:
+
+		<ul>
+			<li>Space efficient storage of backups.
+			<li>Secure, authenticated data encryption.
```
Comment (ThomasWaldmann) [inline index.html]: "data encryption" could be read as "only data, not metadata, encryption".

so leave away "data"? or say "data and metadata"?

File: index.html [hunk 3]
```diff
@@ -1,11 +1,132 @@
...
+			<li>Easy installation on multiple platforms: Linux, macOS, BSD, ...
+			<li>Free software (BSD license).
+			<li>Backed by a large and active open source community.
+		</ul>
+
+		<div id="links">
+			<div class="link">
+				<a href="https://borgbackup.readthedocs.io/en/stable/installation.html">
+					<span class="action">Get</span><br>
+					Borg Backup
```
Comment (ThomasWaldmann) [inline index.html]: sometimes Borg Backup, sometimes BorgBackup...

Comment (ThomasWaldmann) [inline index.html]: I tend to BorgBackup - easier to search online.

Comment (enkore) [inline index.html]: Lets do it consistently. I'll take a look where it needs changing in other places.

**Comment by enkore** [comment]
http://www.borgbackup.org/

**Comment by enkore** [comment]
Org - display name corrected
Docs - nothing to do. _For some reason Google shows the title as "Borg Backup ...", but that's not actually anywhere in the docs._

**Pull request merged**